cmd/serve/serve.go

```diff
@@ -153,4 +155,5 @@ func init() {
 	ServeCmd.Flags().StringVarP(&port, "port", "p", "8080", "Port to run the server on")
 	ServeCmd.Flags().StringVarP(&metricsPort, "metrics-port", "", "8081", "Port to run the metrics-server on")
 	ServeCmd.Flags().StringVarP(&backend, "backend", "b", "openai", "Backend AI provider")
+	ServeCmd.Flags().BoolVarP(&enableRest, "enable-REST", "r", false, "Enable REST api using gppc-gateway")
```

Comment:
Perhaps we can use `--http` flag to enable rest API?

Reply:
updated